end-to-end-tests/fixtures/utils.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2024 PixieBrix, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { type BrowserContext, chromium } from "@playwright/test";
+import { CI, MV, PWDEBUG, SLOWMO } from "../env";
+import path from "node:path";
+
+export const launchPersistentContextForExtension = async (
```

Comment:
Not sure if this is more grammatically correct? We are creating a browser instance with the extension installed
```suggestion
export const launchPersistentContextWithExtension = async (
```